cpp/src/utilities/miscellany.hpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 BlazingDB, Inc.
+ *     Copyright 2018 Eyal Rozenberg <eyalroz@blazingdb.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef UTIL_MISCELLANY_HPP_
+#define UTIL_MISCELLANY_HPP_
+
+#include <cstdlib> // for std::div
+
```

Comment:
Add a file scope Doxygen block to describe what kinds of things go in this file.